<launch>
    <!--node pkg="tf" type="static_transform_publisher" name="world_to_cam" args="0 0 0 -1.57 0 -1.57 camera_odom_frame usb_cam_link 10" /-->
        <!--map usb_cam_link 都是 frame_id. 这里切记 args 里边的结果 x,y,z 以及rpy会累加计算到-->

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1280" /> 
    <param name="image_height" value="720" /> 
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam_link" />
    <param name="io_method" value="mmap"/>
    <param name="autofocus" value="0"/>
  </node>

  <!--node name="image_proc" pkg="image_proc" type="image_proc" output="screen" ns="usb_cam" >
  </node-->
        
    <node name="kcf_detection_node" pkg="uavros_kcffollow_simulation" type="kcf_detection_node.py" respawn="false" output="screen">
        <param name="subscriber" value="/usb_cam/image_raw"/>
        <param name="config" value="camera_param.yaml"/>
    </node>

</launch>
